File: app/src/main/java/be/digitalia/fosdem/db/entities/Bookmark.java
package be.digitalia.fosdem.db.entities;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = Bookmark.TABLE_NAME)
public class Bookmark {

	public static final String TABLE_NAME = "bookmarks";

	@PrimaryKey
	@ColumnInfo(name = "event_id")
	private final long eventId;

	public Bookmark(long eventId) {
		this.eventId = eventId;
	}

	public long getEventId() {
		return eventId;
	}
}

File: app/src/main/kotlin/be/digitalia/fosdem/db/entities/Bookmark.kt
package be.digitalia.fosdem.db.entities

import androidx.room3.ColumnInfo
import androidx.room3.Entity
import androidx.room3.PrimaryKey

@Entity(tableName = Bookmark.TABLE_NAME)
class Bookmark(
        @PrimaryKey
        @ColumnInfo(name = "event_id")
        val eventId: Long
) {
    companion object {
        const val TABLE_NAME = "bookmarks"
    }
}